docs/docs/flink-ddl.md

```diff
@@ -146,7 +146,7 @@ CREATE TABLE `hive_catalog`.`default`.`sample` (
 ) WITH ('format-version'='2');
 ```
 
-Table create commands support the commonly used [Flink create clauses](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/create/) including:
+Table create commands support the commonly used [Flink create clauses](https://nightlies.apache.org/flink/flink-docs-release-{{ defaultFlinkVersion }}/docs/dev/table/sql/create/) including:
```

Comment:
i like the idea to call this `default...version`.
however, i noticed in other places we go without the "default" prefix. is it because they mean different things?

Reply:
I originally used `defaultFlinkVersion` but later switched to `flinkVersionMajor` to be consistent with existing `FLINK_VERSION_MAJOR`. Thanks for catching this.